apps/showcase/src/app/component-replacement/component-replacement.component.ts

```diff
@@ -50,13 +49,11 @@ import {
 })
 export class ComponentReplacementComponent implements AfterViewInit {
   private readonly inPageNavPresService = inject(InPageNavPresService);
-
-  @ViewChildren(InPageNavLinkDirective)
-  private readonly inPageNavLinkDirectives!: QueryList<InPageNavLink>;
+  private readonly inPageNavLinkDirectives = viewChildren<InPageNavLink>(InPageNavLinkDirective);
```

Comment:
Can't we use something like this to have the equivalent as before ?
```suggestion
  private readonly inPageNavLinkDirectives = viewChildren.required<InPageNavLink>(InPageNavLinkDirective);
```